plugins/woocommerce/includes/customizer/class-wc-shop-customizer.php

```diff
@@ -885,6 +885,11 @@ public function has_terms_and_conditions_page_id() {
 	}
 }
 
-if ( ! wc_current_theme_is_fse_theme() ) {
+global $pagenow;
+if (
+	$pagenow === 'customize.php' ||
+	isset( $_REQUEST['customize_theme'] ) || // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	! wc_current_theme_is_fse_theme()
```

Comment:
This condition is probably not needed?

Reply:
Oh NVM, this is needed to make sure with classic themes registration happens early enough to trigger the menu item showing.